learning/features/math/vector_math.rst

```diff
@@ -243,7 +243,7 @@ Facing
 
 We can use this fact to detect whether an object is facing toward another
 object. In the diagram below, the player ``P`` is trying to avoid the
-zombies ``A`` and ``B``. Can the zombies see the player?
+zombies ``A`` and ``B``. Assuming zombies have a field of view of **180°**, can they see the player?
```

Comment:
"have a field of view of" sounds awkward. I would instead use "Assuming a zombie's field of view is **180°**, can it see the player?"

Reply:
Can I use "Assuming a zombie's field of view is 180°, can they see the player?"

Reply:
I'm not sure what the preferred pronoun is for a zombie, so I guess so. :)

Reply:
there's an image with 2 zombies, out of which only one has the player in its FOV.

Reply:
Yes, but the question isn't applying to the group, it's applying to each individual zombie. How about "Assuming each zombie's..." ?